### PR TITLE
chore(consensus): Deduplicate code in {consensus,tecdsa}_performance_test 

### PR DESCRIPTION
**Pull request by kpop-dfinity**
We introduce a new util library to be shared across consensus system tests.
Currently, it only has one function.

### CODE REVIEW

**Comment by cgundy** [comment]
Hey looks like there's a small issue with `lock-generate`, we'll make the fix and you can rebase your PR with master. 👍

**Comment by cgundy** [comment]
The fix has been merged to master. 👍

**Comment by kpop-dfinity** [comment]
> The fix has been merged to master. 👍

Thanks!

**Review by basvandijk — APPROVED** [review]
Great to see more deduplication in system-tests! One step closer to [IDX-3144](https://dfinity.atlassian.net/browse/IDX-3144).

File: rs/tests/consensus/utils/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+use std::time::Duration;
+
+/**
+ * 1. Delete existing tc rules (if present).
+ * 2. Add a root qdisc (queueing discipline) for an htb (hierarchical token bucket).
+ * 3. Add a class with bandwidth limit.
+ * 4. Add a qdisc to introduce latency with jitter.
```
Comment (eichhorl) [inline rs/tests/consensus/utils/src/lib.rs:7]: I think this was removed
```suggestion
 * 4. Add a qdisc to introduce latency.
```

**Review by eichhorl — APPROVED** [review]

File: rs/tests/consensus/utils/src/lib.rs (hunk 1 above)
Comment (kpop-dfinity) [reply rs/tests/consensus/utils/src/lib.rs]: Yes, thanks!

**Pull request merged**